Work with text/src/com/epum/primisen/task_02/parse_text/utill/Regex.java

```diff
@@ -0,0 +1,23 @@
+package com.epum.primisen.task_02.parse_text.utill;
+
+public enum Regex {
+
+    PARAGRAPH("    "),
+    SENTENCE("[\\w\\s]*[^.!?]"),
+    WORD("\\w*[^\\s]"),
+    SYMBOL("\\w");
+
+    private String regex;
+
+    Regex(String regex){
+        this.regex = regex;
+    }
+
+    public String getRegex() {
+        return regex;
+    }
+
+    public void setRegex(String regex) {
```

Comment:
remove setter. It is bad practice to use setter in enum. enums are generally thought of as constant, but an enum with a public field or public setter is not only non-constant, but also vulnerable to malicious code. Ideally fields in an enum are private and set in the constructor